src/views/MainPage/components/OwnOrganization/index.tsx

```diff
@@ -1,23 +1,90 @@
+import Image from 'next/image';
+import { Ref, forwardRef } from 'react';
+import IcArrowLeft from '@src/assets/icons/ic_arrow_left.svg';
+import IcArrowRight from '@src/assets/icons/ic_arrow_right.svg';
+import { useIsMobile } from '@src/hooks/useDevice';
+import useInfiniteCarousel from '@src/hooks/useInfiniteCarousel';
 import { OWN_ORGANIZATION_LIST } from '@src/lib/constants/main';
+import { TextWeightType } from '@src/lib/types/universal';
 import Tab from '../Tab';
 import OwnOrganizationCard from './Card';
 import * as S from './style';
 
-export default function OwnOrganization() {
+type OwnOrganizationCardProps = {
+  nameKor: string;
+  nameEng: string;
+  description: TextWeightType[];
+  frontSideBg: { mo: string; pc: string; ta: string; bigPc: string };
+  backSideBg: string;
+};
+
+function OwnOrganization(_props: unknown, ref: Ref<HTMLDivElement>) {
+  const isMobileSize = useIsMobile('768px');
+  const tab = isMobileSize ? 'Team' : '';
+
+  const {
+    carouselRef,
+    infiniteCarouselList,
+    handleCarouselSwipe,
+    handleTouchStart,
+    handleTouchEnd,
+  } = useInfiniteCarousel<OwnOrganizationCardProps>(OWN_ORGANIZATION_LIST, '(-100% - 20px)');
+
   return (
-    <div>
+    <S.Wrapper id="team" ref={ref}>
       <Tab
-        tab={'(3) 자체 운영 기구'}
-        title={'SOPT를 운영하는 자체 기구'}
+        tab={tab}
+        title={'솝트가 운영하는 자체 기구'}
         description={
-          'SOPT에는 솝트를 자체적으로 운영하는 두 가지의 기구가 존재합니다. 한 기수 이상 활동한\n사람들이 모여, 솝트가 보다 유연하고 열정적인 경험으로 채워질 수 있도록 노력하죠.'
+          '솝트에는 자체적으로 운영하는 네 가지의 팀이 있습니다.\n솝트의 보다 유연하고 열정적인 경험을 위해 노력하죠.'
         }
       />
-      <S.Wrapper>
-        {OWN_ORGANIZATION_LIST.map((organization) => (
-          <OwnOrganizationCard key={organization.nameEng} {...organization} />
-        ))}
-      </S.Wrapper>
-    </div>
+      {!isMobileSize && (
+        <S.ContentWrapper>
+          {OWN_ORGANIZATION_LIST.map((organization) => (
+            <OwnOrganizationCard key={organization.nameEng} {...organization} />
+          ))}
+        </S.ContentWrapper>
+      )}
+      {isMobileSize && (
```

Comment:
사소한 거지만 여기 그냥 삼항연산자로 정리해줄 수 있을 것 같아요!! 
```suggestion
      {!isMobileSize ? (
        <S.ContentWrapper>
          {OWN_ORGANIZATION_LIST.map((organization) => (
            <OwnOrganizationCard key={organization.nameEng} {...organization} />
          ))}
        </S.ContentWrapper>
      )}
      : (
```

Reply:
삼항 연산자 자체의 이점이 있다기보다 
우리가 보통 
```js
if (bool) { true일때 로직 }
if (!bool) { false일때 로직 }
```
이렇게 구현하지 않고 
```js
if (bool) { true일때 로직 }
else { false일때 로직 }
```
일반적으로 이런 식으로 구현하니까 이런 맥락으로 제안드린 의견입니다 ! ! 혹시 어떻게 생각하시나욥

Reply:
오,, 저는 
위의 방식으로 구현을 하는 편입니다,,!
이유는 if - else 문의 사용을 지양하자는 글을 읽어서 그런데요
[참고자료1](https://html-jc.tistory.com/473)
[참고자료2](https://opendeveloper.tistory.com/entry/%EC%BD%94%EB%93%9C%EC%9D%B4%EC%8A%88-if%EB%AC%B8%EC%97%90%EC%84%9C-else%EC%99%80-else-if%EB%A5%BC-%EC%A7%80%EC%96%91%ED%95%98%EB%8A%94-%EC%9D%98%EA%B2%AC%EC%97%90-%EB%8C%80%ED%95%9C-%EC%83%9D%EA%B0%81%EC%A0%95%EB%A6%AC)
물론 굳이 그럴 필요 없다는 주장도 있어서 이에 대해서는 좀 더 공부를 해야할 필요가 있어 보이긴 하지만
어찌댔든 저는
확실히 어떤 경우에 실행문이 실행되는지 알 수 있는
if(~)
if(!~)
를 사용하는 편입니다,,!!
근데 사실 진짜 저는 상관 없어서 그런데
삼항 연산자로 표현하는 것이 더 보기 편하시다 하면 그렇게 해도 좋습니다!!
(그 에러 페이지에서도 그렇게 하신 거 이미 봤었고요,,!)
어떻게 생각하시나요?? 진짜 저는 상관 없습니다,,,!!!!!!!! 진짜 찐짜 진짜 편하게 의견 주세요

Reply:
좋은데요?